### DISCUSSION TITLE
I've Used AppMover Script for AVR Logger

### DISCUSSION THREAD

**Discussion by eineb01** [007revad/Synology_app_mover#99] [General]
Hello Dave,
Today I moved the AVRLogger app to an NVME volume with AppMover version 3.0.59. Since then, AvrLogger has continued to work without any problems; all of the parameters I recorded remained are fully working. So you can expand the table for which apps AppMover script works with AvrLogger as working.

Greatings eineb01

**Comment by 007revad** [comment]
Thanks @eineb01
